### PR TITLE
Fix #1334

### LINKED ISSUE #1334: Missing two error codes
* [ ] https://github.com/fastify/fastify/blob/1ad553df56f6833048a7e105fb500ac6db60ce2f/lib/reply.js#L65
* [ ] https://github.com/fastify/fastify/blob/1ad553df56f6833048a7e105fb500ac6db60ce2f/lib/wrapThenable.js#L19

I propose we fix those before 2.0.0 is out.

### PR DESCRIPTION
**Pull request by 0xsuid**
Added two Missing error codes in files ``` lib/reply.js ``` and ``` lib/wrapThenable.js ``` which fix issue #1334

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by 0xsuid** [comment]
I am not sure about change made to hooks.test.js so kindly review it.

**Comment by 0xsuid** [comment]
``` t.is(err.message, 'You cannot use `send` inside the `onError` hook') ```
Should it changed to be t.pass() ?? @delvedor

**Comment by delvedor** [comment]
@0xsuid Nope, you can leave it as is :)

File: test/hooks.test.js [hunk 1]
```diff
@@ -2015,7 +2015,7 @@ test('reply.send should throw if called inside the onError hook', t => {
       reply.send()
       t.fail('Should throw')
     } catch (err) {
-      t.is(err.message, 'You cannot use `send` inside the `onError` hook')
+      t.is(err.message, 'FST_ERR_SEND_INSIDE_ONERR: You cannot use `send` inside the `onError` hook')
```
Comment (mcollina) [inline test/hooks.test.js]: Can you add an assertion here `t.is(err.code, 'FST_ERR_SEND_INSIDE_ONERR')`?

**Comment by mcollina** [comment]
I've just added a nit.

**Comment by 0xsuid** [comment]
Test updated :) @delvedor @mcollina

**Comment by 0xsuid** [comment]
Error On Linux node_8_x :confused: 
From Line 3141:
```
                                     # Subtest: file option   
2018-12-23T13:55:02.6524332Z         1..13  
2018-12-23T13:55:02.6524482Z         ok 1 - should not error  
2018-12-23T13:55:02.6524634Z         ok 2 - expect truthy value  
2018-12-23T13:55:02.6524860Z         not ok 3 - ENOENT: no such file or directory, open '/tmp/sonic-boom-5094-424,796640439-0'  
2018-12-23T13:55:02.6525012Z           ---  
2018-12-23T13:55:02.6525163Z           errno: -2  
2018-12-23T13:55:02.6525220Z           code: ENOENT  
2018-12-23T13:55:02.6525255Z           syscall: open  
2018-12-23T13:55:02.6525432Z           path: '/tmp/sonic-boom-5094-424,796640439-0'  
2018-12-23T13:55:02.6525495Z           test: file option  
2018-12-23T13:55:02.6525529Z           ...  
2018-12-23T13:55:02.6525561Z           
2018-12-23T13:55:02.6525611Z         # test count(3) != plan(13)  
2018-12-23T13:55:02.6525650Z         # failed 1 of 3 tests  
2018-12-23T13:55:02.6525815Z     not ok 21 - file option # time=20.103ms  
```
@delvedor @mcollina

**Comment by mcollina** [comment]
restarted

**Comment by 0xsuid** [comment]
So, what should we do now? @mcollina @delvedor

**Comment by mcollina** [comment]
@0xsuid I'm rerunning CI again now.

**Comment by mcollina** [comment]
CI was flaky, landing.

**Pull request merged**